frontend/public/lib/selectOptionsOperations.js

```diff
@@ -1,6 +1,11 @@
 export function parseOptions(options, parentPropertyName) {
+  //Return an array of parent properties with their subcategories. If the parent property of an item is missing: assume it's a top level property
```

Comment:
Thanks for adding this!
```suggestion
  // Returns an array of parent properties with their subcategories. If the parent property of an item is missing, assume it's a top-level property
```